sql/src/main/java/io/crate/analyze/TableParameters.java

```diff
@@ -110,8 +110,7 @@
      */
     static final Set<Setting> SETTINGS_NOT_INCLUDED_IN_DEFAULT = Set.of(
         IndexMetaData.INDEX_NUMBER_OF_ROUTING_SHARDS_SETTING,
-        IndexService.GLOBAL_CHECKPOINT_SYNC_INTERVAL_SETTING,
-        IndexModule.NODE_STORE_ALLOW_MMAP
```

Comment:
Isn't this still needed here so that nodes with older versions won't complain e.g. while new partitions are created?

Reply:
I am not sure. It is a node setting, but not a table parameter, so the new and old partitions won't be aware of this setting.

Reply:
right, its a node setting so irrelevant here.